Frameworks/include/Starboard.h

```diff
@@ -205,4 +205,19 @@ inline CAPoint3D operator*(float f, const CAPoint3D& v) {
     return CAPoint3D(v.x * f, v.y * f, v.z * f);
 }
 
+// Wrapper for C++/WinRT types when you need a trivial default constructor
```

Comment:
>C++/WinRT types when you need a trivial default constructo [](start = 15, length = 58)

Out of curiosity, do you know why c++/winrt doesn't provide this?  Seems odd that clients would have to provide this.